scripts/plugins/openvswitch-config-update

```diff
@@ -199,16 +190,12 @@ def update(session, args):
         if new_controller and fail_mode_changed and pool_fail_mode == "secure":
             oc = vswitchCfgQuery(["get", "Bridge", bridge, "other-config"])
             m = re.match('.*hwaddr="([0-9a-fA-F:].*)".*', oc)
-            if m and m.group(1) in pool_mgmt_macs.keys():
+            if m and m.group(1) in pool_mgmt_macs:
                 mgmt_bridge = bridge
                 host_mgmt_mac = m.group(1)
                 host_mgmt_device = pool_mgmt_macs[host_mgmt_mac]
 
-    if (
-        host_mgmt_mac is not None
-        and mgmt_bridge is not None
-        and host_mgmt_device is not None
-    ):
+    if all([host_mgmt_mac, mgmt_bridge, host_mgmt_device]):
```

Comment:
While the checks are not equivalent, it's better to check for falseness in this case since the empty values would also be incorrect

Reply:
Re-run vswitch testcase,  job (3876591) passed